media_driver/linux/gen9/ddi/media_sku_wa_g9.cpp

```diff
@@ -540,6 +546,8 @@ static bool InitGlkMediaSku(struct GfxDeviceInfo *devInfo,
 
     MEDIA_WR_SKU(skuTable, FtrHDR, 1);
 
+    MEDIA_WR_SKU(skuTable, FtrUseSwSwizzling, 1);
```

Comment:
GLK is small core , is using map_gtt, when this WA take effect?

Reply:
Used before mapping
https://github.com/intel/media-driver/blob/3f754f70437309a95682368d43e2c6708631f1b3/media_driver/linux/common/ddi/media_libva_util.cpp#L1209